src/libraries/System.Collections.NonGeneric/src/System.Collections.NonGeneric.csproj

```diff
@@ -21,6 +21,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="System.Memory" />
```

Comment:
These are not required until we re-add the keyword, let me know if I should remove them for now.